code/datums/gameticker.dm

```diff
@@ -117,7 +117,7 @@ var/global/current_state = GAME_STATE_WORLD_INIT
 	switch(master_mode)
 		if("random","secret") src.mode = config.pick_random_mode()
 		if("action") src.mode = config.pick_mode(pick("nuclear","wizard","blob"))
-		if("intrigue") src.mode = config.pick_mode(pick("mixed_rp", "traitor","changeling","vampire","conspiracy","spy_theft","arcfiend", prob(50); "extended"))
+		if("intrigue") src.mode = config.pick_mode(pick(prob(300);"mixed_rp", prob(150); "traitor","changeling","vampire", prob(50); "conspiracy", "spy_theft","arcfiend", prob(50); "extended"))
```

Comment:
Overall admin opinion seems to be that traitor could be at 200.

Reply:
Also probably changeling at 75 or so as changelings tend to make people salty a lot.

Reply:
Input written down